.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,43 @@
+name: Feature Request
+description: Suggestions for new or different behavior.
+title: 'feat: '
+labels:
+  - enhancement
+  - needs triage
+  - question
```

Comment:
ok in hindsight having `question` and `needs triage` is probably unnecessary lol but again your call here

```suggestion
```